### ISSUE TITLE
HTTPS Support

### ISSUE THREAD

**Issue by chris37879** [alefeuvre/foundation-grid-displayer#15]
Instead of loading everything with http:// consider using // instead, similar to the way Google's CDN recommends things be loaded, as it stands the bookmarklet won't work on https:// sites.

**Comment by alefeuvre** [comment]
Good idea Chris. I'll do the update as soon as I can, i.e. it may take a while as I'm off travelling at the moment. Do not hesitate to submit a pull request if you want the change to be done sooner.

-----Original Message-----
From: Chris Hendrickson
Sent: 22/05/2014 13:21
To: alefeuvre/foundation-grid-displayer
Subject: [foundation-grid-displayer] HTTPS Support (#15)

Instead of loading everything with http:// consider using // instead, similar to the way Google's CDN recommends things be loaded, as it stands the bookmarklet won't work on https:// sites.

**Comment by chris37879** [comment]
If I find the time this weekend I may, I just noticed it and wanted to make
sure other devs were aware.

**Comment by alefeuvre** [comment]
Hi Chris,
Last week I released the version 3.2 of the bookmarklet. It comes with HTTPS support. No need to re-add the bookmarklet to get the new version.
Thanks again for suggesting this enhancement!

**Comment by alefeuvre** [comment]
Following [this post on Stack Overflow](http://stackoverflow.com/questions/24646171/grid-displayer-bookmarklet-for-bootstrap-3-stopped-working), I'm afraid I had to rollback to a version with no HTTPS support. I now have to look for a solution that supports both the HTTPS and FILE protocols. Sorry about that!

**Comment by chrismendis** [comment]
@alefeuvre Instead of relying on relative URLs, we could parse the protocol from `window.location`. If this solution sounds appealing, I'd be interested in implementing it.

**Comment by chrismendis** [comment]
@alefeuvre of course, if the parsed protocol was anything other than `http` or `https` then we would use `http`.

**Comment by chrismendis** [comment]
@alefeuvre WIP PR started: https://github.com/alefeuvre/foundation-grid-displayer/pull/17.

**Comment by doggy8088** [comment]
@alefeuvre I think the HTTPS support is more relevant than FILE support at this time.  Are you consider support HTTPS now?
